tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +46,71 @@ public void chown() throws IOException, AlluxioException {
+    clearLoginUser();
+    GroupMappingServiceTestUtils.resetCache();
+    Configuration.set(PropertyKey.SECURITY_GROUP_MAPPING_CLASS,
+        IdentityUserGroupsMapping.class.getName());
+    Configuration.set(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE.getAuthName());
+    Configuration.set(PropertyKey.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true");
+
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    String nonexistUser = "nonexistuser";
+    List<String> groups = CommonUtils.getGroups(newOwner);
+    String nonexistGroup = "nonexistgroup";
+    Random rand = new Random();
+    int numOfGroups = groups.size();
+    // chown legalUser:legalGroup
+    for (int i = 0; i < 8; i++) {
+      try {
+        int k = rand.nextInt(numOfGroups);
+        String group = groups.get(k);
+        mFsShell.run("chown", newOwner + ":" + group, "/testFile");
+        String currentGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+        Assert.assertEquals(group, currentGroup);
+      } catch (IllegalArgumentException e) {
+        mOldOutput.format("The user %s does not belong any group.%n", newOwner);
+      }
+    }
+
+    String originalOwner = mFileSystem.getStatus(new AlluxioURI("/testFile")).getOwner();
+    String originalGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+    // chown illegalUser:legalGroup
+    for (int i = 0; i < 8; i++) {
+      try {
+        int k = rand.nextInt(numOfGroups);
+        String group = groups.get(k);
+        mFsShell.run("chown", nonexistUser + ":" + group, "/testFile");
+        String currentOwner = mFileSystem.getStatus(new AlluxioURI("/testFile")).getOwner();
+        String currentGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+        Assert.assertEquals(originalOwner, currentOwner);
+        Assert.assertEquals(originalGroup, currentGroup);
+      } catch (IllegalArgumentException e) {
+        mOldOutput.format("The user %s does not belong to any group.%n", newOwner);
+      }
+    }
+
+    // chown illegalUser:illegalGroup
+    mFsShell.run("chown", nonexistUser + ":" + nonexistGroup, "/testFile");
+    do {
+      String currentOwner = mFileSystem.getStatus(new AlluxioURI("/testFile")).getOwner();
+      String currentGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+      Assert.assertEquals(originalOwner, currentOwner);
+      Assert.assertEquals(originalGroup, currentGroup);
+    } while (false);
+
+    // chown legalUser:illegalGroup
+    mFsShell.run("chown", newOwner + ":" + nonexistGroup, "/testFile");
+    do {
+      String currentOwner = mFileSystem.getStatus(new AlluxioURI("/testFile")).getOwner();
+      String currentGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+      Assert.assertEquals(originalOwner, currentOwner);
+      Assert.assertEquals(originalGroup, currentGroup);
```

Comment:
nit: can you extract this original owner and group checking to a helper function? It is duplicated for multiple times.